src/irq/irq.go

```diff
@@ -206,3 +226,47 @@ func matchesRegex(value, pattern string) bool {
 	match, err := regexp.MatchString(pattern, value)
 	return err == nil && match
 }
+
+func logChanges(changed, managed []int, cpus string) {
+	if len(managed) > 0 {
+		log.Printf("Skipped read-only (managed?) IRQs: %s: input/output error",
+			cpulists.GenCPUlist(managed))
```

Comment:
At this point in the code, we can't assume the error is always "input/output error", without putting a mechanism to ensure the corresponding part of the code stays the same.
Also this is extra info.
```suggestion
		log.Printf("Skipped managed IRQs: %s", cpulists.GenCPUlist(managed))
```

Reply:
Fixed, thanks! 45e5cea